Assignment/solution_student/portfolio.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Portfolio</title>
+	<link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+	<nav class="nav text-center">
+		<a href="aboutme.html" class="nav-link">About Me</a> |
+		<a href="portfolio.html" class="nav-link nav-link-active">Portfolio</a>
+	</nav>
+	<h1 class="text-center">Joe Johnson</h1>
+	<hr>
+	<h2>Experience</h2>
+        
+		<p>I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an <span class="bold-italic">awesome</span> Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.</p>
+    
+	<h4>Portfolio</h4>
+    
+	<ul>
+		<li>
+			<p class="margin-none font-sans-serif"><span class="bold-italic">Web Developer</span>, Relaxr, <span class="font-small">2015 - present</span></p>
+
+			<p>Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.</p>
+
+			<img src="images/relaxr_deliverable.png" alt="Relaxr Website" align="center" class="image-border"></li>
+
+		<li>
+			<p class="margin-none font-sans-serif"><span class="bold-italic">Web Developer</span>, Startup Matchmaker, <span class="font-small">2015 - present</span></p>
+
+			<p>Used a design team's wireframes to develop this company's responsive homepage.</p>
+
+			<img src="images/startup_matchmaker_deliverable.png" alt="Matchmaker Website" align="center" class="image-border"></li>
+
+		<li>
+			<p class="margin-none font-sans-serif"><span class="bold-italic">Web Developer</span>, CitiPix, <span class="font-small">2015 - present</span></p>
+
+			<p>Built web app prototype that allows users to store and quickly retrieve photos using keywords.</p>
+
+			<img src="images/citipix_deliverable.png" alt="Citipix Website" align="center" class="image-border"></li>
+	</ul>
+
+	<h4>Education</h4>
+
+	<ul>
+		<li class="font-sans-serif font-size-ul"><span class="bold-italic">General Assembly</span> - FEWD 2015</li>
+
+		<li class="font-sans-serif font-size-ul"><span class="bold-italic">College of Soft Knocks</span> - B.S. 2009</li>
+
+		<li class="font-sans-serif font-size-ul"><span class="bold-italic">School of Hard Knocks</span> - 2005 (Honors)</li>
+		<br>
+		<br>
```

Comment:
Avoid using the `<br>` tag to space things out — only use it if you actually have to break a line (so it should always be in a `<p>` or a `<heading>` etc). Instead use the `padding` and `margin` properties to achieve this. You could give the `<ul>` a class of "education-list" or something, and then add a bottom `margin` or `padding`